vaadin-contextmenu-event.html

```diff
@@ -43,7 +43,7 @@
         this.info.touchJob = Polymer.Debounce(this.info.touchJob, function() {
           var t = Polymer.Gestures.findOriginalTarget(e);
           var ct = e.changedTouches[0];
-          if (this.info._ios) {
+          if (this.info._ios && !e.shiftKey) {
```

Comment:
Interesting, thanks for the link. I don't believe it's used in real life https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/shiftKey but still you're technically correct here.